packages/addons/sveltekit-adapter/index.ts

```diff
@@ -0,0 +1,100 @@
+import { defineAddon, defineAddonOptions } from '@sveltejs/cli-core';
+import { exports, functions, imports, object, type AstTypes } from '@sveltejs/cli-core/js';
+import { parseJson, parseScript } from '@sveltejs/cli-core/parsers';
+
+type Adapter = {
+	id: string;
+	package: string;
+	version: string;
+};
+
+const adapters: Adapter[] = [
+	{ id: 'node', package: '@sveltejs/adapter-node', version: '^5.2.9' },
+	{ id: 'static', package: '@sveltejs/adapter-static', version: '^3.0.6' },
+	{ id: 'vercel', package: '@sveltejs/adapter-vercel', version: '^5.5.0' },
+	{ id: 'cloudflare-pages', package: '@sveltejs/adapter-cloudflare', version: '^4.8.0' },
+	{ id: 'cloudflare-workers', package: '@sveltejs/adapter-cloudflare-workers', version: '^2.6.0' },
+	{ id: 'netlify', package: '@sveltejs/adapter-netlify', version: '^4.4.0' }
+];
```

Comment:
Did you had any specific ordering in mind? I would have expected the same ordering as on the website: https://svelte.dev/docs/kit/adapters.

Seems to be mostly ordered by popularity (expect netlify): https://npmcharts.com/compare/@sveltejs/adapter-node,@sveltejs/adapter-static,@sveltejs/adapter-vercel,@sveltejs/adapter-cloudflare,@sveltejs/adapter-cloudflare-workers,@sveltejs/adapter-netlify

Should there also be an option to adapter auto? I think that would be strange and we should avoid it.